lldb/source/Symbol/SwiftASTContext.cpp

```diff
@@ -2034,39 +2034,49 @@ lldb::TypeSystemSP SwiftASTContext::CreateInstance(lldb::LanguageType language,
 
   // Now if the user fully specified the triple, let that override the one
   // we got from executable's options:
```

Comment:
Do this comment need to stay next to the `IsFullySpecifiedTriple` check, as they were before this change?

Also just curious, where is this user specified triple coming from?

Reply:
My understanding is that this comment refers to CreateTargetWithFileAndTargetTriple / Arch https://lldb.llvm.org/cpp_reference/classlldb_1_1SBDebugger.html#af53e525e77a729938fd25587dc8bab81